griptape/drivers/image_query/base_image_query_driver.py

```diff
@@ -16,6 +16,7 @@
 @define
 class BaseImageQueryDriver(SerializableMixin, ExponentialBackoffMixin, ABC):
     structure: Optional[Structure] = field(default=None, kw_only=True)
+    max_output_tokens: int = field(default=256, kw_only=True, metadata={"serializable": True})
```

Comment:
Lets keep it named as `max_tokens` for consistency with other Drivers.

Reply:
Agreed that `max_output_tokens` is probably a better field name, but the other PR was only in the context of the Tokenizers, not the Drivers.

Reply:
All the other drivers have inputs as text, not images. so to clarify my question: are the other drivers that have `max_tokens` intended to be for the output tokens or the input tokens?

Reply:
Updated back to `max_tokens`